File: VSC/1360.number-of-days-between-two-dates.java
/*
 * @lc app=leetcode id=1360 lang=java
 *
 * [1360] Number of Days Between Two Dates
 */

// @lc code=start
class Solution {
    public int daysBetweenDates(String date1, String date2) {
        return Math.abs(daysFrom1971(date1) - daysFrom1971(date2));
    }
}
// @lc code=end

File: 1360.number-of-days-between-two-dates.py
#
# @lc app=leetcode id=1360 lang=python3
#
# [1360] Number of Days Between Two Dates
#

# @lc code=start
class Solution:
    def daysBetweenDates(self, date1: str, date2: str) -> int:
        from datetime import datetime
        d1 = datetime.strptime(date1, '%Y-%m-%d')
        d2 = datetime.strptime(date2, '%Y-%m-%d')
        return abs((d1 - d2).days)
# @lc code=end
